docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -68,11 +68,35 @@ The GDC Data Portal has been completely revamped to include the following featur
 
 ### Known Issues and Workarounds
 
+* Cohorts:
+    * Cohorts are under active development and their behavior may change in the first several months after the release of GDC Portal 2.0. As this process may result in the loss of saved cohorts on the portal, we highly recommend [exporting cohorts](quick_start.md#main-toolbar) locally.
```

Comment:
I can't find any release notes examples, but I think the link needs to be /Data_Portal/Users_Guide/quick_start.md#main-toolbar

Reply:
oh good point. i forgot i was writing release notes and not user's guide... :P will fix.